public/admin/config.yml

```diff
@@ -644,3 +644,70 @@ collections:
           - label: Metadata description
             name: metadataDescription
             widget: text
+
+  - label: Locations
+    name: locations
+    label_singular: Location
+    description: description
+    format: json
+    create: true
+    files:
+      - label: States
+        name: states
+        file: src/cms-content/locations/state.json
+        fields:
+          - label: Regions
+            name: regions
+            widget: list
+            fields:
+              - label: Location Name
+                name: locationName
+                widget: string
+              - label: Location Type
+                name: locationType
+                widget: select
+                options: 
+                  - { label: "State", value: "state" }
+              - label: FIPS Code
+                name: fips
+                widget: string
+
+  - label: Vaccination Information
+    name: vaccinationInfo
+    label_singular: Region Vaccine information
+    description: description
+    format: json
+    files:
+      - name: Vaccine Information-State
+        label: State Vaccine Information 
+        file: public/cms-content/vaccines/state.json
+        fields:
+          - label: Regions
+            name: regions
+            widget: list
+            fields:
+              - label: Location Name
+                name: locationName
+                widget: relation
+                collection: locations
+                value_field: regions.*.locationName
+                file_name: states.json
+                search_fields: ["regions.*.locationName"]
+              - label: Fips Code
+                name: fips
+                widget: relation
+                collection: locations
+                file_name: states.json
+                value_field: regions.*.fips
+                search_fields: ["regions.*.locationName", "regions.*.fips"]
+              - label: Hidden
+                name: hidden
+                widget: boolean
+                required: false
+              - label: Eligibility Information URL
+                name: eligibilityInfoUrl
+                widget: string
```

Comment:
I would make `eligibilityInfoUrl` required, there isn't much point on adding the data item without at least this

Reply:
I think the default value is true, so it shouldn't be necessary